### PR TITLE
fix tests

### PR DESCRIPTION
**Pull request by ihatespawn**
some tests weren't working because the link was invalid or like with loom, the feature was unsupported but the test excepted a download

if #604 is also merged, every test should be working good... not including the b23.tv shortlink, i don't know how to create it.

### CODE REVIEW

File: src/util/tests.json [hunk 1]
```diff
@@ -947,7 +947,7 @@
     }],
     "streamable": [{
         "name": "regular video",
-        "url": "https://streamable.com/03r3c2",
+        "url": "https://streamable.com/p7blpp",
```
Comment (dumbmoron) [inline src/util/tests.json:950]: it might be a better idea to use some older streamable video that is still up, because they purge videos after some time

Comment (ihatespawn) [reply src/util/tests.json:950]: 2 months isn't old enough for you?

Comment (dumbmoron) [reply src/util/tests.json:950]: no because they purge videos after 90 days

Comment (ihatespawn) [reply src/util/tests.json:950]: ok, i'll update the link in a sec

**Review by ihatespawn — COMMENTED** [review]
this is an official video from ~7 months ago

**Review by dumbmoron — APPROVED** [review]

**Pull request merged**